src/python/pants/backend/project_info/peek_test.py

```diff
@@ -22,23 +24,42 @@
             id="null-case",
         ),
         pytest.param(
-            [Files({"sources": []}, Address("example", target_name="files_target"))],
+            [
+                TargetData(
+                    Files({"sources": ["*.txt"]}, Address("example", target_name="files_target")),
+                    ("foo.txt", "bar.txt"),
+                    tuple(),
+                )
+            ],
             True,
             dedent(
                 """\
                 [
                   {
                     "address": "example:files_target",
                     "target_type": "files",
-                    "sources": []
+                    "sources": [
+                      "*.txt"
+                    ],
+                    "expanded_sources": [
+                      "foo.txt",
+                      "bar.txt"
+                    ],
+                    "expanded_dependencies": []
```

Comment:
Sorry for not participating in the bikeshed yesterday (and consequently, feel free to completely ignore this), but I would invert this to `dependencies_raw` (the literal deps) and `dependencies` (the expanded deps), as the latter is much more likely to be useful to a consumer, and it would be confusing to accidentally consume the former without knowing the difference.

Reply:
I think I agree with that. While I like the principal of "the `peek` key is the same as the field name"...I think it's more intuitive for users to see `dependencies` and `dependencies_raw`.

I also thought about `raw_dependencies`, but I think `dependencies_raw` is better.